src/features/bookOffer/components/Calendar/Calendar.tsx

```diff
@@ -70,6 +71,14 @@ export const getMinAvailableDate = (markedDates: MarkedDates): string | undefine
   )[0]
 }
 
+export const getDayDescription = (price: number, hasSeveralPrices?: boolean) => {
+  let dayDescription = ''
+  if (hasSeveralPrices) dayDescription += 'dès '
+  dayDescription += formatToFrenchDecimal(price).replace(' ', '')
+
+  return dayDescription
+}
```

Comment:
```suggestion
export const getDayDescription = (price: number, hasSeveralPrices?: boolean) => {
  return `${hasSeveralPrices ? 'dès ' : ''} {formatToFrenchDecimal(price).replace(' ', '')}`
}
```

Je ne sais pas si c'est mieux.
Ou alors 
```suggestion
export const getDayDescription = (price: number, hasSeveralPrices?: boolean) => {
  let dayDescription = hasSeveralPrices ? 'dès ' : ''
  dayDescription += formatToFrenchDecimal(price).replace(' ', '')

  return dayDescription
}
```

Je n'aime pas les déclarations vides, souvent ça indique que quelque chose peut être fait autrement.
Je pense que la seconde solution que je te propose est un bon compromis.